### PR TITLE
Location is your permanent residence

### PR DESCRIPTION
**Pull request by dmurko**

### CODE REVIEW

File: 5_People/salary-system.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ Once a year, in January, during our [bi-annual IRL](https://blog.niteo.co/irls/)
 
 ### Location Factor
 
-Location factor takes into account the living costs. We determine them by comparing Numbeo's [cost of living between San Francisco and the Nitean's city](https://www.numbeo.com/cost-of-living/compare_cities.jsp?country1=United+States&city1=San+Francisco%2C+CA&country2=Germany&city2=Berlin) and calculating the *Numbeo Location Ratio*.
+Location factor takes into account the living costs. We determine them by comparing Numbeo's [cost of living between San Francisco and the Nitean's city](https://www.numbeo.com/cost-of-living/compare_cities.jsp?country1=United+States&city1=San+Francisco%2C+CA&country2=Germany&city2=Berlin) and calculating the *Numbeo Location Ratio*. For the purposes of the Salary System, we take your official permanent residence (address stated in your passport), and do not change it for seasonal relocations or longer travel.
```
Comment (zupo) [inline 5_People/salary-system.md]: What about temporary residence? If I am not mistaken I can have a temporary residence in my passport? Or is this just a construct of epic Slovenian bureaucracy?

Comment (zupo) [inline 5_People/salary-system.md]: What about temporary residence (`zacasno prebivalisce`)? If I am not mistaken I can have a temporary residence in my passport? Or is this just a construct of epic Slovenian bureaucracy? 

http://www.mnz.gov.si/mnz_za_vas/osebni_dokumenti_in_prebivalisce/prijava_in_odjava_prebivalisca/zacasno_prebivalisce/

Comment (zupo) [reply 5_People/salary-system.md]: http://www.mnz.gov.si/mnz_za_vas/osebni_dokumenti_in_prebivalisce/prijava_in_odjava_prebivalisca/zacasno_prebivalisce/

Comment (zupo) [reply 5_People/salary-system.md]: Seems this does not affect "address in passport". So, meh.

**Comment by zupo** [comment]
Missing a sentence about "what if my town is not in Numbeo".

**Comment by zupo** [comment]
Should we limit the "closest city in Numbeo" to a city that has X amount of entries in Numbeo, representing "good enough" data?

**Comment by dmurko** [comment]
That sounds like a good idea, yes.

**Comment by dmurko** [comment]
@zupo What's should be the minimum? LJ has 252 entries for the last 18 months, Celje, for example, just 25.

**Comment by zupo** [comment]
@dmurko: huh, no idea. 100 in the last 12 months?

**Comment by dmurko** [comment]
Their metric is 18 months, so we can have a 100 for that. Already written like that. Just review and approve.

**Comment by zupo** [comment]
100 in 18 months sound quite low. I can probably get 10 friends to submit data that skews the Numbeo factor considerably. 

250 is better than 100. Shall we put the minimum at 250? 

What if a a big city in a different country is closer than a big city in your own country?

**Comment by dmurko** [comment]
@zupo That barely covers Ljubljana (252). I'd say 180 in 18 months and we'll probably miss smaller cities than LJ, esp in non-western countries.

And we always look for the city in the same country. Since taxes, infrastructure, expenses etc. are more related to the country than geographical location, it has to be the same country.

**Comment by zupo** [comment]
OK, let's do 180 in 18 months.

**Pull request merged**